### PR TITLE
STOEGE-19591 _helpers.tpl & app-cm.yaml refactoring

### PR DESCRIPTION
**Pull request by SuslikDaRete**

### CODE REVIEW

**Review by manifest — APPROVED** [review]

File: chart/templates/app-cm.yaml [hunk 1]
```diff
@@ -180,13 +178,13 @@ data:
     {{- end }}
     {{- if eq "local" .type }}
     trusted = [
-      {{- range $account_label := .trusted }}
-        {{ $ns_audience | list $account_label | join "." | quote }},
+      {{- range $accountLabel := .trusted }}
+        {{ list $accountLabel $tenantObjectAudience | join "." | quote }},
```
Comment (manifest) [inline chart/templates/app-cm.yaml:182]: `$tenantObjectAudience` has to be `.Values.app.svc.audience`

**Review by manifest — CHANGES_REQUESTED** [review]

File: chart/templates/_helpers.tpl [hunk 2]
```diff
@@ -51,35 +51,35 @@ app.kubernetes.io/instance: {{ .Release.Name }}
...
 */}}
-{{- define "dispatcher.shortNamespaceSetting" -}}
-{{- $shortns := regexSplit "-" .Release.Namespace -1 | first }}
-{{- if has $shortns (list "production" "p") }}
+{{- define "dispatcher.tenantObjectAudience" -}}
+{{- $vars := index . 0 -}}
+{{- $tenant := index . 1 -}}
+{{- $env := regexSplit "-" $vars.Release.Namespace -1 | first -}}
+{{- $devEnv := ""}}
+{{- if eq $env "p" }}
```
Comment (manifest) [inline chart/templates/_helpers.tpl:77]: Shouldn't we return `$tenant` in this case?

File: chart/templates/app-cm.yaml (hunk 1 above)
Comment (SuslikDaRete) [reply chart/templates/app-cm.yaml]: Yeap, you're right
Fixed

File: chart/templates/_helpers.tpl (hunk 2 above)
Comment (SuslikDaRete) [reply chart/templates/_helpers.tpl]: Not here. Rewrote the condition to make it clearer

**Review by manifest — APPROVED** [review]

**Review by manifest — APPROVED** [review]

**Pull request merged**